tests/test_export_mermaid.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests for exporting schemapacks in mermaid format."""
+
+import pytest
+
+from schemapack import load_schemapack
+from schemapack.export.mermaid import export_mermaid
+from tests.fixtures.examples import EXAMPLES_DIR, VALID_SCHEMAPACK_PATHS
+
+
+@pytest.mark.parametrize(
+    "with_properties, file_suffix",
+    [(True, "_w_props.mm.txt"), (False, "_wo_props.mm.txt")],
+)
+def test_export_mermaid(with_properties: bool, file_suffix: str):
```

Comment:
How do we deal with additionalProperties being set to true? Maybe add a test for the expected behavior as well.

Reply:
Good point, I've added an asterisk row for that case and added it to the test schema as well. 
<img width="194" alt="Screenshot 2024-04-16 at 22 28 50" src="https://github.com/ghga-de/schemapack/assets/5948880/66bc90eb-11a0-4b98-b9d9-dc0047e82fd0">